R/Table.R

```diff
@@ -70,7 +70,7 @@ odbc_write_table <-
     fieldDetails <- tryCatch({
       details <- odbcConnectionColumns(conn, name)
       details$param_index <- match(details$name, names(values))
-      details[!is.na(details$param_index), ]
+      details[!is.na(details$param_index) & !is.na(details&data_type), ]
```

Comment:
```suggestion
      details[!is.na(details$param_index) & !is.na(details$data_type), ]
```
?

Reply:
Ugh - sorry about that.
Thanks!

Reply:
Don't worry, spotting this sort of problem makes me feel useful 😄